src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/PendingBucketDeletionComponent.scala

```diff
@@ -8,7 +8,7 @@ trait PendingBucketDeletionComponent {
   import driver.api._
 
   class PendingBucketDeletions(tag: Tag) extends Table[PendingBucketDeletion](tag, "BUCKET_DELETION") {
```

Comment:
I'm not a fan of this naming convention: too close to the `PendingBucketDeletion` case class.  What about `PendingBucketDeletionTable` for the `Table` and `pendingBucketDeletionQuery` for the `TableQuery`?